pkg/core/object/object.go

```diff
@@ -1,70 +1,18 @@
 package object
 
 import (
-	objectV2 "github.com/nspcc-dev/neofs-api-go/v2/object"
-	"github.com/nspcc-dev/neofs-api-go/v2/refs"
 	"github.com/nspcc-dev/neofs-sdk-go/object"
 	addressSDK "github.com/nspcc-dev/neofs-sdk-go/object/address"
 )
 
-// Object represents the NeoFS object.
-//
-// Object inherits object type from NeoFS SDK.
-// It is used to implement some useful methods and functions
-// for convenient processing of an object by a node.
-type Object struct {
-	*object.Object
-}
-
-// Address returns address of the object.
-func (o *Object) Address() *addressSDK.Address {
-	if o != nil {
-		aV2 := new(refs.Address)
-		aV2.SetObjectID(o.ID().ToV2())
-		aV2.SetContainerID(o.ContainerID().ToV2())
-
-		return addressSDK.NewAddressFromV2(aV2)
-	}
-
-	return nil
-}
-
-// SDK returns NeoFS SDK object instance.
-func (o *Object) SDK() *object.Object {
-	if o != nil {
-		return o.Object
-	}
-
-	return nil
-}
-
-// NewFromV2 constructs Object instance from v2 Object message.
-func NewFromV2(obj *objectV2.Object) *Object {
-	return &Object{
-		Object: object.NewFromV2(obj),
-	}
-}
-
-// NewFromSDK constructs Object instance from NeoFS SDK Object.
-func NewFromSDK(obj *object.Object) *Object {
-	return &Object{
-		Object: obj,
-	}
-}
-
-// New constructs blank Object instance.
-func New() *Object {
-	return NewFromSDK(object.New())
-}
+// AddressFromObject returns address of the object.
+func AddressFromObject(obj *object.Object) *addressSDK.Address {
```

Comment:
What about having an `Address()` in SDK?

Reply:
Also, consider `AdressOf` as a name.

Reply:
> What about having an `Address()` in SDK?

I don't mind though it seems rather redundant due to its simplicity. In `node` it is very often used (as seen in current PR), but I have not yet seen a single use case in other projects.
More thoughts? @alexvanin @carpawell @KirillovDenis 

And I agree that `AddressOf` is more neat.

Reply:
> I have not yet seen a single use case in other projects.

Client applications usually don't fetch address from object, indeed. Server applications (custom storage \ relay node implementations) can definitely benefit from that, but SDK is more client application oriented as for now.